.gitignore

```diff
@@ -1,2 +1,5 @@
 deployment
 omw-installations
+plugin/openmissilewars-plugin.iml
+plugin/.idea/*
+testing/*
```

Comment:
Would prefer not to have this.

Reply:
Hmm, then should I ignore IDE files (without deleting them)?

Reply:
To ignore non-omw files add them to `.git/info/exclude`